### DISCUSSION TITLE
ViGEmBus won't fix the error for me

### DISCUSSION THREAD

**Discussion by yoylemixil** [nefarius/ViGEmBus#122] [Support]
If ViGEmBus won't fix the error for me, I will start to get very rage

**Comment by Masamune3210** [comment]
Please go chew on a tree, nobody is here to jump at your call. All it's going to do is piss people off and get you banned from issues

**Reply by nefarius** [reply]
Just banned this kid 🤣

**Reply by Masamune3210** [reply]
Thank the heavens, now they can get "very rage" in peace

**Reply by Masamune3210** [reply]
They apparently made issues in two other projects, and they both use about as much logic as this one lol

**Reply by nefarius** [reply]
I mean what do you expect from a username only chosen by a pre-pubescent Call of Duty player 😝

**Reply by Masamune3210** [reply]
Fair lol
